packages/twenty-front/src/utils/create-required-context.ts

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+
+export const createRequiredContext = <TContext>(name: string) => {
```

Comment:
No need to pass the name if it's only to debug it. The name could be outdated when we refactoring + it's easy to find the bad context usage in stack trace